src/components/MoviesList/MoviesList.js

```diff
@@ -1,16 +1,15 @@
+import { useSelector } from "react-redux";
 import React from "react";
-import { useContext } from "react";
 
 import "./MoviesList.css";
 import MovieCard from "../MovieCard/MovieCard";
-import { AppContext } from "../../hooks/AppContext";
 
 const MoviesList = () => {
-  const { moviesContext } = useContext(AppContext);
+  const movies = useSelector((state) => state.movies.movies);
   return (
     <>
       <ul className="movie-list">
-        {moviesContext.movies.map((movie, index) => (
+        {movies.map((movie, index) => (
           <li key={index}>
```

Comment:
Usage of index as key is not recommended https://reactjs.org/docs/lists-and-keys.html#:~:text=We%20don't%20recommend%20using,an%20index%20as%20a%20key.

Reply:
Good point. I changed index for movie id. Thanks!